projects/1.16/assets/dynamic-trees-byg/dtbyg/lang/zh_cn.json

```
@@ -172,10 +172,10 @@
     "item.dtbyg.orange_birch_seed": "橙色桦树絮",
     "item.dtbyg.red_birch_seed": "红色桦树絮",
     "item.dtbyg.yellow_birch_seed": "黄色桦树絮",
-    "item.dtbyg.brown_oak_seed": "棕色橡实",
-    "item.dtbyg.orange_oak_seed": "橙色橡实",
-    "item.dtbyg.red_oak_seed": "红色橡实",
-    "item.dtbyg.joshua_seed": "约书亚种子",
+    "item.dtbyg.brown_oak_seed": "棕色橡子",
+    "item.dtbyg.orange_oak_seed": "橙色橡子",
+    "item.dtbyg.red_oak_seed": "红色橡子",
+    "item.dtbyg.joshua_seed": "短叶丝兰种子",

```

| **审查级别** |
|--------------|
| ℹ️ 信息 / 不阻止合并  |
[短叶丝兰 - 维基百科，自由的百科全书](https://zh.wikipedia.org/wiki/%E7%9F%AD%E8%91%89%E7%B5%B2%E8%98%AD)
[Yucca brevifolia - Wikipedia](https://en.wikipedia.org/wiki/Yucca_brevifolia)
```suggestion
    "item.dtbyg.joshua_seed": "短叶丝兰种子",
```
原文：**约书亚种子**

---

反正约书亚种子是错的